analysis/analysis.go

```diff
@@ -483,6 +482,37 @@ func assessMetricStatus(metric v1alpha1.Metric, result v1alpha1.MetricResult, te
 	return v1alpha1.AnalysisPhaseRunning
 }
 
+func assessMetricFailureInconclusiveOrError(metric v1alpha1.Metric, result v1alpha1.MetricResult) (v1alpha1.AnalysisPhase, []string) {
+	var msg1, msg2 string
```

Comment:
I don't think we need msg1 since it always consists of `assessed ` plus the phase.  Methods that call `assessMetricFailureInconclusiveOrError` already have that phase information.